libs/ojsonnet/Value_jsonnet.ml

```diff
@@ -1,74 +1,68 @@
-(* Yoann Padioleau
- *
- * Copyright (C) 2022 r2c
- *
- * This library is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public License
- * version 2.1 as published by the Free Software Foundation, with the
- * special exception on linking described in file LICENSE.
- *
- * This library is distributed in the hope that it will be useful, but
- * WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the file
- * LICENSE for more details.
- *)
-module A = AST_jsonnet
-module C = Core_jsonnet
-
 (*****************************************************************************)
-(* Prelude *)
+(* Env *)
 (*****************************************************************************)
-(* Jsonnet "values".
- *
- * See https://jsonnet.org/ref/spec.html#jsonnet_values
```

Comment:
please restore this toplevel comment too.
Each of our file has this Copyright then Prelude comments, so good to be consistent.